### ISSUE TITLE
[Supporting the Platform - Make Code - Extensions] Focus indicator is not visible on controls in high contrast theme under 'Extensions' screen.

### ISSUE THREAD

**Issue by sonimishra** [microsoft/monaco-editor#1533]
**User Experience:**
User will find difficulty to see any control if keyboard focus is not visible on any interactive control in high contrast theme as they won't be able to take an action on such control and also they will surely miss the functionality and will proceed further.

**Test Environment:**
Version 1903 (OS Build 18867.1000)
Platform: Edge, Chrome

**Pre-Requisite:**
Turn on Windows High contrast theme (Settings->Ease of access->High Contrast->Turn on High Contrast->Select High Contrast White and Black

**Repro Steps:**
1. Launch application URL : https://makecode.microbit.org/#
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "Extensions" option and select it.
7. Navigate through all the controls except 'learn more' link appearing under "Extensions" page
8. Verify focus visibility on controls like (Bluetooth services, Devices, Radio-broadcast, Servo etc.) after applying high contrast theme(Black/White).

**Actual Result:**
Keyboard focus is not visible on controls like (Bluetooth services, Devices, Radio-broadcast, Servo etc.) link after applying high contrast theme(Black/White) under 'Extensions' screen and due to this visual impaired user is unable to see the keyboard focus on that control and also he/she is unable to perform functionality over such control.
 
**Expected Result:**
Keyboard focus should be visible on controls like (Bluetooth services, Devices, Radio-broadcast, Servo etc.) link after applying high contrast theme(Black/White) under 'Extensions' screen,so that visual impaired user could easily see the keyboard focus on that control to perform functionality over such control.

**Note:** 
This issue also exists with Chrome.

**MAS Reference:**
[MAS 4.3.1 - No Disruption of Accessibility Features](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={672c8a8b-ed4f-463f-868f-e8131caf9b74})

[Accessibility Insights!] - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**Comment by sonimishra** [comment]
Bug logged in incorrect path. Hence closing this bug.
